docs/moduleAna3AIMEC.rst

```diff
@@ -87,79 +96,101 @@ The following figure illustrates the raster transformation process.
             In the new coordinate system (s,l)
 
 
-Here is the definition of the different indicators and outputs from the AIMEC post-processing process:
+Here is the definition of the different indicators and outputs from the AIMEC
+post-processing process:
 
 Mean and max values along path
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-All two dimensional field results (for example peak velocities / pressure or flow thickness) can be projected into the curvilinear system using
-the previously described method. The maximum and average values of those fields are computed in each cross-section (l direction).
-For example the maximum :math:`A_{cross}^{max}(s)` and average :math:`\bar{A}_{cross}(s)` of the two
-dimensional distribution :math:`A(s,l)` is:
+All two dimensional field results (for example peak velocities / pressure or flow thickness) can be
+projected into the curvilinear system using the previously described method. The maximum and
+average values of those fields are computed in each cross-section (l direction).
+For example the maximum :math:`A_{cross}^{max}(s)` and average :math:`\bar{A}_{cross}(s)` of
+the two dimensional distribution :math:`A(s,l)` is:
 
 .. math::
     A_{cross}^{max}(s) = \max_{\forall l \in [-\frac{w}{2},\frac{w}{2}]} A(s,l) \quad\mbox{and}\quad
     \bar{A}_{cross}(s) = \frac{1}{w}\int_{-\frac{w}{2}}^{\frac{w}{2}} A(s,l)dl
 
 Runout point
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~
+
+The runout point is always given in regards to a peak result field (:math:`A(s,l)` which could be
+peak pressure of flow thickness...) and a threshold value (:math:`A_{lim}>0`).
+The runout point (:math:`s=s_{runout}`) and the related :math:`(x_{runout},y_{runout})`
+in the original coordinate system, corresponds to the last point in flow direction where the
+chosen peak result :math:`A_{cross}^{max}(s)` is above the threshold value :math:`A_{lim}`.
 
-The runout point corresponding to a given pressure threshold :math:`P_{lim}>0kPa` is the first point :math:`s=s_{runout}`
-where the maximum peak pressure falls below the pressure limit (:math:`P_{cross}^{max}(s)<P_{Lim}`). This :math:`s=s_{runout}` is related
-to a :math:`(x_{runout},y_{runout})` in the original coordinate system. It is very important to note that the position of this
-point depends on the chosen pressure limit value. It would also be possible to use :math:`\bar{P}_{cross}(s)<P_{Lim}` instead of
-:math:`P_{cross}^{max}(s)<P_{Lim}`.
+.. Note:: It is very important to note that the position of the runout point depends on the chosen
+          threshold value and peak result field. It would also be possible to use
+          :math:`\bar{A}_{cross}(s)>A_{lim}` instead of :math:`A_{cross}^{max}(s)>A_{lim}`
```

Comment:
Would 'It is also possible' describe it better than 'it would also be possible'?

Reply:
Yes we can use it. Then I kind of understand that there is an option to change this somewhere in the ini files...